<ui version="4.0" >
 <class>ICatAdvancedSearch</class>
 <widget class="QWidget" name="ICatAdvancedSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ICat Advanced Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QLabel" name="advsearchtitleLabel" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>Advanced Investigations Search</string>
     </property>
     <property name="indent" >
      <number>321</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="advframeWidget" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>207</height>
      </size>
     </property>
     <property name="styleSheet" >
      <string notr="true" >background-image: url(:/BackGroundImage/ICatSearchBackGround.png);</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="invstNameLabel" >
          <property name="text" >
           <string>Investigation Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="investigatonNameEdit" >
          <property name="maximumSize" >
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="runNumberLabel" >
          <property name="text" >
           <string>Run#:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLineEdit" name="startRunEdit" >
          <property name="maximumSize" >
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Start Run Number</string>
          </property>
          <property name="inputMask" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <widget class="QLineEdit" name="endRunEdit" >
          <property name="maximumSize" >
           <size>
            <width>81</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>End Run Number</string>
          </property>
          <property name="inputMask" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="invstAbstractLabel" >
          <property name="text" >
           <string>Investigation Abstract:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="invstAbstractEdit" >
          <property name="maximumSize" >
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="startdateLabel" >
          <property name="text" >
           <string>Start Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLineEdit" name="startdateLineEdit" >
          <property name="maximumSize" >
           <size>
            <width>101</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Date in DD/MM/YYY format</string>
          </property>
          <property name="inputMask" >
           <string>99/99/9999;_</string>
          </property>
          <property name="text" >
           <string>//</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" >
         <widget class="QToolButton" name="startdatetoolButton" >
          <property name="maximumSize" >
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="sampleLabel" >
          <property name="text" >
           <string>Sample:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="sampleEdit" >
          <property name="maximumSize" >
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="enddateLabel" >
          <property name="text" >
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QLineEdit" name="enddateLineEdit" >
          <property name="maximumSize" >
           <size>
            <width>101</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Date in DD/MM/YYY format</string>
          </property>
          <property name="inputMask" >
           <string>99/99/9999;_</string>
          </property>
          <property name="text" >
           <string>//</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" >
         <widget class="QToolButton" name="enddatetoolButton" >
          <property name="maximumSize" >
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="invstsurnameLabel" >
          <property name="text" >
           <string>Investigators surname:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="invstsurnameEdit" >
          <property name="maximumSize" >
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QLabel" name="instrumentLabel" >
          <property name="text" >
           <string>Instrument:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QComboBox" name="instrumentBox" >
          <property name="maximumSize" >
           <size>
            <width>101</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Instrument Box</string>
          </property>
          <property name="styleSheet" >
           <string notr="true" >QComboBox#instrumentBox QListView{background-color: white;background-image: url(ICatComboBackground.png);background-attachment: scroll;}QComboBox#instrumentBox QListView QScrollBar:vertical{background-image: url(:/images/ICatComboVScrollbar.png); background-repeat: repeat-y; width: 17px; height:20px;} </string>
          </property>
         </widget>
        </item>
        <item row="3" column="4" >
         <widget class="QLabel" name="keywordsLabel" >
          <property name="text" >
           <string>    KeyWord(s):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5" >
         <widget class="QLineEdit" name="keywordsEdit" >
          <property name="maximumSize" >
           <size>
            <width>121</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="datafilenameLabel" >
          <property name="text" >
           <string>Datafile name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLineEdit" name="datafilenameEdit" >
          <property name="maximumSize" >
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="2" >
         <widget class="QLabel" name="invstTypeLabel" >
          <property name="text" >
           <string>Investigation Type:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" >
         <widget class="QComboBox" name="invstTypeBox" >
          <property name="maximumSize" >
           <size>
            <width>101</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet" >
           <string notr="true" >QComboBox#invstTypeBox QListView{background-color: white;background-image: url(ICatComboBackground.png);background-attachment: scroll;}QComboBox#invstTypeBox QListView QScrollBar:vertical{background-image: url(:/images/ICatComboVScrollbar.png); background-repeat: repeat-y; width: 17px; height:20px;} </string>
          </property>
         </widget>
        </item>
        <item row="4" column="4" >
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>78</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="5" >
         <widget class="QCheckBox" name="casesensitiveBox" >
          <property name="text" >
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>587</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton" >
          <property name="toolTip" >
           <string>Search Investigations</string>
          </property>
          <property name="text" >
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="advsearchLabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="advSearchtableWidget" >
     <property name="styleSheet" >
      <string notr="true" >background-color: rgb(216, 225, 255);</string>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>568</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>helpButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>advSearchtableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ICatSearchBackGround.qrc" />
 </resources>
 <connections/>
</ui>
